security/tools/jwt/README.md

```diff
@@ -0,0 +1,61 @@
+# Generate Json Web Tokens Signed by Google Service Account
+
+The python script (sa-jwt.py) provided here allows the user to generate a JWT signed
+by a Google service account.
+
+## Before you start
+
+- Run the following command to install python dependences.
+```
+  pip install google-cloud
+```
+
+- Create a service account or use an existing service account, and download the service account private key.
+
+  - In the [Service Accounts page](https://console.cloud.google.com/iam-admin/serviceaccounts),
+  click CREATE SERVICE ACCOUNT, or select one of the existing service accounts.
+  - Click "Create Key" from the drop-down menu, and select the default JSON key type. The key file
+    will automatically downloads to your computer.
+
+## Usage
+
+The script expects two arguments, the audience ("aud") and the service account key file ("service_account_file"). Optionally,
+you can also set the the issue name. By default, the issuer name and the "sub" claim are set to the service account email.
+
+Type the following command to see the help message.
+
+```
+  python sa-jwt.py -h
+```
+
+It outputs the following:
+```
+usage: sa-jwt.py [-h] [-iss ISSUER] aud service_account_file
+
+Python script generates a JWT signed by a Google service account
+
+positional arguments:
+  aud                   Audience.
```

Comment:
Can this be optional too? And comma separated for multi entries (not important if it adds to much complexity)

Reply:
Made aud claim optional.